lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -1017,23 +1018,37 @@ bool swift::diagnoseNonSendableTypesInReference(
   // For functions, check the parameter and result types.
   SubstitutionMap subs = declRef.getSubstitutions();
   if (auto function = dyn_cast<AbstractFunctionDecl>(declRef.getDecl())) {
-    for (auto param : *function->getParameters()) {
-      Type paramType = param->getInterfaceType().subst(subs);
-      if (diagnoseNonSendableTypes(
-              paramType, fromDC, loc, diag::non_sendable_param_type,
-              (unsigned)reason, function->getDescriptiveKind(),
-              function->getName(), getActorIsolation()))
-        return true;
+    switch (funcCheckType) {
+    case FunctionCheckType::Results: break;
```

Comment:
typically we put the break on the line below the case to make it easier to visually distinguish cases that fall-through or not.